author/views.py

```diff
@@ -1,3 +1,13 @@
-from django.shortcuts import render
+from django.views import generic
+from .models import Author
 
-# Create your views here.
+class BloggerDetailView(generic.DetailView):
+    model = Author
```

Comment:
We have to use queryset instead of model. 
and try to  use selected_related and prefetch_related concept